### PR TITLE
feat: redesign profile

### LINKED ISSUE #566: ⚡ Feature | PROFILE - story tabs
Display a tab row on top of the stories.
The line should be 1px color 6 full width.
The first tab to display is "stories". Display the number of stories the user published too.
Capture d’écran 2022-06-22 à 18 07 05

Figma component: https://www.figma.com/file/uPVMeBq0bERyViiRMXzLN8/Sigle-design-system?node-id=1826%3A1658

### PR DESCRIPTION
**Pull request by gregogun**
* Adds **stories** tab to profile page
* Adds copyable stx address to bio
* Adds username tag next to display name
* Updates visual layout of writer info

Closes #566 #583

### CODE REVIEW

File: sigle/src/ui/IconButton.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export const IconButton = styled('button', {
 
   '&:disabled': {
     opacity: 0.5,
-    pointerEvents: 'none',
+    cursor: 'not-allowed',
```
Comment (gregogun) [inline sigle/src/ui/IconButton.ts:11]: with pointerEvents set to to none, the tooltip message 'Copied!' fails to show once the copy button is clicked

File: .changeset/seven-shrimps-design.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@sigle/app': patch
```
Comment (pradel) [inline .changeset/seven-shrimps-design.md:2]: this can be minor

File: sigle/src/modules/publicHome/components/PublicHome.tsx [hunk 3]
```diff
@@ -232,32 +266,109 @@ export const PublicHome = ({ file, settings, userInfo }: PublicHomeProps) => {
...
+
+                  '&:hover': {
+                    color: '$gray10',
+                  },
+                  '&:active': {
+                    color: '$gray12',
+                  },
+                }}
+              >
+                {abbreviateAddress(userInfo.address)}
```
Comment (pradel) [inline sigle/src/modules/publicHome/components/PublicHome.tsx:296]: I think this link should be clickable and open a new tab showing the address in the stacks explorer, wdyt?

File: sigle/src/modules/publicHome/components/PublicHome.tsx [hunk 4]
```diff
@@ -232,32 +266,109 @@ export const PublicHome = ({ file, settings, userInfo }: PublicHomeProps) => {
...
+                '&:hover': {
+                  '& button': {
+                    display: 'block',
+                  },
+                },
+              }}
+            >
+              <Typography
+                size="subheading"
+                css={{
```
Comment (pradel) [inline sigle/src/modules/publicHome/components/PublicHome.tsx:285]: as this css is being shared among all the items in this row, maybe it make sense to create a styled component at the beginning of this file?

File: sigle/src/modules/publicHome/components/PublicHome.tsx [hunk 5]
```diff
@@ -112,6 +133,16 @@ export const PublicHome = ({ file, settings, userInfo }: PublicHomeProps) => {
     unfollowUser({ userFollowing, address: userInfo.address });
   };
 
+  const handleClick = () => {
```
Comment (pradel) [inline sigle/src/modules/publicHome/components/PublicHome.tsx:136]: Let's rename this one to smth like handleCopyAddress ?

**Review by pradel — CHANGES_REQUESTED** [review]

**Review by pradel — APPROVED** [review]

**Pull request merged**